magefiles/utils.go

```diff
@@ -439,3 +440,34 @@ func HandleErrorWithAlert(err error, errLevel slack.ErrorSeverityLevel) error {
 	}
 	return nil
 }
+
+func getChangeFiles() (rulesengine.Files, error) {
+	var gitDiff = sh.OutCmd("git", "diff")
+	output, err := gitDiff("--name-status", "main..HEAD")
+
+	if err != nil {
+		klog.Error("Failed to run git status locally")
+		return nil, err
+	}
+	var changes rulesengine.Files
+	var file rulesengine.File
+	for _, line := range strings.Split(output, "\n") {
+
+		fileAttr := strings.Split(line, "\t")
+		if string(fileAttr[0]) == "??" {
```

Comment:
This is a hold over from when I was originally using a form of `git status` rather than `git diff`. The status command was returning unstaged changes with `??`. I'll remove this piece of code. Done!